### DISCUSSION TITLE
Welcome to Hands-On-Intelligent-Agents-with-OpenAI-Gym Discussions!

### DISCUSSION THREAD

**Discussion by praveen-palanisamy** [PacktPublishing/Hands-On-Intelligent-Agents-with-OpenAI-Gym#31] [General]
## 👋 Welcome to the HOIAWOG Community!
  This is a place where you can connect with the author, other readers and audience for the ["Hands-On Intelligent Agents With Openai Gym" book](https://www.packtpub.com/big-data-and-business-intelligence/hands-intelligent-agents-openai-gym) and [code](https://github.com/PacktPublishing/Hands-On-Intelligent-Agents-with-OpenAI-Gym):
  * Ask any questions related to the book or code that you’re wondering about.
  * Showcase your own creations and customization.
  * Share your thoughts on how you are using the book/code with others.
  * Welcome others and be kind :). Remember that this is a community we build together 💪.

  You can get started by commenting below with an introduction of yourself and tell us what you are working on.

**Comment by tsadigovAgmail** [comment]
Hello I am a software dev/consultant. I am interested in AI related topics more as a hobby.
I read through the book and try to get my head around. Focus on chapter 6 - deep Q learning.
I am not sure if I can make sense of it. I want to experiment and compare its effects. Built a desktop PC for thease endeavors :)

After I could not understand correlation between my changes and the results, I decided to have a baseline so I  run the example without it learning anything. To my surprise I had very similar chart. Now I dont know if the learning does any god here.

Started discussion on that
https://github.com/PacktPublishing/Hands-On-Intelligent-Agents-with-OpenAI-Gym/discussions/32

**Reply by praveen-palanisamy** [reply]
@tsadigovAgmail, Welcome to the HOIAWOG community!
Nice to know about your background and your interest in AI topics! 

I will be happy to help you make more sense of Chapter 6 (and others) and help provide clarity to your experiment investigations. I have replied to your post  #32 and we shall continue our discussion on it. Rest assured, the RL agent learns to do/act better!
